### DISCUSSION TITLE
how does "mod crit type on hitgroup" work.

### DISCUSSION THREAD

**Discussion by nameme1moretime** [nosoop/SM-TFCustomAttributeStarterPack#65] [Q&A]
like what does the attribute line have to look like, cause i want the head shots to work.
[XCR.txt](https://github.com/nosoop/SM-TFCustomAttributeStarterPack/files/8937307/XCR.txt)

**Comment by nosoop** [comment]
[The `value` line in this CW3 config has an example](https://github.com/nosoop/SM-TFCustomAttributeStarterPack/blob/c5c3abbb659fb467157b6c1815d11bdbf2856efd/configs/customweapons/magnum_opus.cfg#L19):

```
"mod crit type on hitgroup"
{
	"plugin"	"custom-attribute-adapter/mod_crit_type_on_hitgroup"
	"value"		"hitgroup=1 crit_type=2"
}
```

**Reply by nameme1moretime** [reply]
thanks
